src/app/_components/PageHeader.tsx

```diff
@@ -22,9 +22,13 @@ type TitleProps = {
   children: string
 }
 
+type PageHeaderImageProps = (StaticImageProps | ImageProps) & {
+  objectFit?: ImageObjectFit
```

Comment:
Is it optional?

Reply:
Yes, passing nothing should be possible. Most `PageHeader` don't need `objectFit` for their image.